### DISCUSSION TITLE
签名服务器冻结/封号状况调查

### DISCUSSION THREAD

**Discussion by SuperKenVery** [Mrs4s/go-cqhttp#2270] [Polls]
本人有点害怕被封号，出于自己的怂和给大家提供信息，希望在这里做一个冻结/封号相关的调查。感谢尝螃蟹的人！

**Comment by xia0ji233** [comment]
不知道各位冻结qq有没有这种情况，在输入身份证和姓名人脸验证的时候，那个下一步点不了

**Reply by SkyYeXianer** [reply]
我发现了这个问题，是IP的问题，挂服务器的得用服务器的IP地址，挂梯子得用梯子的IP地址。同这个问题一同存在还有注销时也检测IP了。

**Comment by zhzhongshi** [comment]
冻结一次直接90天（

**Comment by skilittle** [comment]
被冻了无法自助解冻，找了客服被要求写保证书

**Reply by SuperKenVery** [reply]
！哈人

**Comment by buger404** [comment]
我这边的情况是冻结一次解冻之后就非常稳定了

**Comment by ghost** [comment]
连续冻结多次，最后冻结90天。

**Comment by chzhong** [comment]
我挂着两个号，很长时间内一直没有被冻结。
3月中旬B号冻结过一次，6月底也就是上一轮大面积 `code 40` / `code 45` 在上签名服务器后4小时马上两个号都被冻结，然后前几天同样又被冻结。

我怀疑和我挂多个号有关，今天对比了一下 `device.json` 发现自动生成的 Mac 地址和 IP 都是一样的。所以停了一个手动改了两者后重新登了。我用的是 docker 跑的，最初的最初 (v1.0.0-rc4 之前）是可以 `docker start -ai` 来完成扫码、验证操作的，更新到 v1.0.0-rc5 后突然间不行了，所以当时就把文件复制到宿主机登录后把 `session.token` 复制回去。最新的 1.1.0 好像又可以在 docker 内直接完成了。不知道这次能持续多久。

**Comment by Fader282** [comment]
换了qsign之后立马被冻结，而且要求人脸验证的时候一直提示光线过暗。

**Comment by rfuy** [comment]
换了qsign之后立马被冻结

**Comment by SuperKenVery** [comment]
7月18挂到现在 挺稳的（

dev分支配合1.1.4qsign
